lib/api/v3/queries/schemas/query_schema_representer.rb

```diff
@@ -105,6 +105,13 @@ def self.filters_schema
                  has_default: true,
                  visibility: false
 
+          schema :timeline_zoom_level,
+                 type: 'String',
+                 required: false,
+                 writable: true,
+                 has_default: true,
+                 visibility: false
```

Comment:
I have no intention to use these values in the frontend (as there is special meaning as to what each unit means, e.g., which grid is going to be rendered), and thus suggest to not implement the allowed values.